gdraw/gdraw.c

```diff
@@ -36,6 +36,9 @@
 #  include <sys/select.h>
 #endif
 
+#if __FreeBSD__ || __NetBSD__ || __OpenBSD__ || __DragonFly__
+#  include <sys/select.h>
```

Comment:
It seems that it would make sense to combine this with the `__Mac` test given that that part of the Macintosh software is also BSD and that the resulting action is the same.

Reply:
I put BSDs apart on purpose.  While MacOS does mimic BSDs here and there, there are more differences than similarities between them.  For instance, display rendering APIs are completely different, etc.  Such grouping prevents complex splits of preprocessor definitions in the future.

Reply:
BTW, I have no idea why Github merged these two successive commits into the same pull request, this was not my intent.

Reply:
@vvp2014, if you push to the same branch, it ends up in the pull request associated with that branch (as long as the pull request is closed).

Reply:
Do you expect further divergent actions within these conditionals? Unless there is a likely imminent change in that direction, this provisioning seems excessive, particularly since it looks so much like an oversight. If we merge it like this, we'll probably get another pull request in the next year merging the two statements, and I'll be the one on the hook to defend this design decision.

Reply:
Well, in my experience grouping platforms according to similarity is justified.  I am not going to argue about that.  At any rate this fix is the only one that is required for building a complete FF on the current OpenBSD.  Please proceed as you please.

Reply:
Thanks for the branching tip!

Reply:
@jtanx, this bothers me but not very much, and I'm inclined to merge in the form proposed by @vvp2014. What do you think?

Reply:
No, just combine the checks into the same ifdef above, *if* at some point they start to differ then we'll deal with that then.

Reply:
@vvp2014, we cannot modify your branch. Would you consolidate the changes so that we can get this merged?

Reply:
All right then, I am going to close this and submit two separate requests, as I intended.